### DISCUSSION TITLE
Live Feed on Web app

### DISCUSSION THREAD

**Discussion by hoffadg** [chuot/rdio-scanner#326] [Q&A] [answered]
Is it possible to enable the live feed by default on the web app?  I know you can in the Android app (paid version).

**Comment by chuot** [comment, marked as answer]
Not possible due to browser restriction. To activate an audioContext, a user interaction must occur prior to fire it up.

**Comment by mitch-kazel** [comment]
Thanks, chuot, for your excellent project.  I found a Chome brower extension that will force the live feed **on** when the web page loads.  Not sure whether I'm allowed to mention the name here.
